addons/pinniped/config-controller/controllers/handlers.go

```diff
@@ -100,3 +94,43 @@ func (c *PinnipedV3Controller) withPackageName(packageName string) builder.Predi
 		},
 	)
 }
+
+func (c *PinnipedV1Controller) addonSecretToCluster(o client.Object) []ctrl.Request {
+	log := c.Log.WithValues(secretNamespaceLogKey, o.GetNamespace(), secretNameLogKey, o.GetName())
+
+	clusterName, labelExists := o.GetLabels()[tkgClusterNameLabel]
+
+	if !labelExists || clusterName == "" {
+		log.Error(nil, "cluster name label not found on resource")
```

Comment:
Shall we log specifically so that we know exactly which cluster or resource is in question?

Reply:
we actually add the name/namespace to the log a few lines above this (line 99), but since we only use that log variable in one spot, I'll just move it down to this line.

Reply:
nevermind.  `WithValues` above got this.

Reply:
I updated it to do it in the same spot since it's less code... also found a place where I added them 2x in the handler below, so updating that as well.